lib/grandpa/message.go

```diff
@@ -77,21 +77,55 @@ func (v *VoteMessage) ToConsensusMessage() (*ConsensusMessage, error) {
 	}, nil
 }
 
-// NeighbourMessage represents a network-level neighbour message
-type NeighbourMessage struct {
-	Version byte
-	Round   uint64
-	SetID   uint64
-	Number  uint32
+// VersionedNeighbourMessage represents the enum of nighbor messages
+type VersionedNeighbourMessage scale.VaryingDataType
```

Comment:
```suggestion
type VersionedNeighbourPacket scale.VaryingDataType
```
Can we start just matching the substrate names.  It'll make it much easier to read both codebases.